linalg/amgxsolver.hpp

```diff
@@ -160,16 +197,19 @@ class AmgXSolver : public Solver
       two iterations of an AMG V cycle with AmgX's default smoother (block
       Jacobi).
 
-      As a solver the preconditioned conjugate gradient method is used. The AMG
-      V-cycle with a block Jacobi smoother is used as a preconditioner.
+      When configured as a solver the preconditioned conjugate gradient method
+      is used with the AMG V-cycle with a block Jacobi smoother is used as a
```

Comment:
minor: I find this a bit hard to read. Maybe "with the AMG V-cycle with a block Jacobi smoother is used as a preconditioner" ->  "with the AMG V-cycle **and** a block Jacobi smoother as preconditioner." ?